README.md

```diff
@@ -2,30 +2,40 @@
 
 # Node.js Nikita
 
-Nikita gather a set of functions usually used during system deployment.
+Nikita gathers a set of functions commonly used during system deployment.
 Documentation is available on the [project website](https://nikita.js.org).
 
 ## Main features 
 
-* Consistent Usage   
+* Consistent Usage
```

Comment:
Could you please remove the uppercase in the second and following words (the `U` in this case shall be `u`)